scripts/parts.py

```diff
@@ -30,8 +30,9 @@
                         dest='mapr_cols', default='all')
     parser.add_argument('--levels-testing', help="Set to test filename if correlator levels not accessible [levels.tst]", 
                         dest='levels_testing', default='levels.tst')
-    parser.add_argument('--date', help="MM/DD/YY or now [now]", default='now')
-    parser.add_argument('--time', help="hh:mm or now [now]", default='now')
+    parser.add_argument('--date', help="UTC YYYY/M/D or now [now]", default='now')
+    parser.add_argument('--time', help="UTC hh:mm or now [now]", default='now')
```

Comment:
This is the 3rd place these exact lines are added. I also see them in args_test.py. I freely admit that I'm not sure how this all works, but can you reduce the code replication by using args_test here?